modules/ROOT/pages/batch-reference.adoc

```diff
@@ -0,0 +1,193 @@
+
+By default, the history of batch job instances persists for 7 days. A monitoring process automatically removes that history when the period expires.
+
+image::batch-job-fields-history-studio.png[alt=Batch Job History fields in Studio,width=500px]
+
+The following table describes each field and includes the XML attribute for the field.
+
+[%header, cols="1a,1a,3a"]
+|===
+| Name | XML | Description
+
+| Max Age | `maxAge` | Maximum age of job instances before they expire, for example: `expiration maxAge="10"`. Default: `maxAge=7`.
+
+| Time Unit | `ageUnit` | The unit of time to apply to the `maxAge`. Valid values: `DAYS`, `HOURS`, `MILLISECONDS`, `MINUTES`, `SECONDS`. Default: `ageUnit=DAYS`.
+
+|===
+
+Note that these fields are attributes to the `<batch:expiration/>` element, which is embedded within `<batch:history/>`:
+
+[source,xml,linenums]
+----
+<batch:job ...>
+  <batch:history >
+    <batch:expiration maxAge="10" ageUnit="MINUTES" />
+  </batch:history>
+  <batch:process-records>
+    <batch:step />
+  </batch:process-records>
+<batch:job />
+----
+
+The XML example is edited to focus on settings in the batch history fields.
+
+[[ref_batch_step_component]]
+== Batch Step Component (<batch:step/>)
+
+The Batch Step component runs during the xref:batch-processing-concept.adoc#process[Process phase] of a batch job instance, not during the Load and Dispatch or On Complete phases.
+
+A minimum of one Batch Step component is required within a Batch Job component. In addition to changing the name of a Batch Step component, you can configure _filters_ that determine which records the component accepts for processing within the component. Any processors that you add to the component act on records that the Batch Job component accepts.
+
+image::batch-step-fields-studio.png[alt=Batch Step fields in Studio,width=500px]
+
+The following table describes each field and includes the XML attribute for the field.
+
+[%header, cols="1,1,3a"]
+|===
+| Field Name | XML | Description
+
+| Name | `name` | Configurable name for a Batch Step component. The default is `Batch_Step` for the first Batch Step component within a Batch Job component. Subsequent steps append a unique number to the name, following the sequence `Batch_Step1`, `Batch_Step2`, and so on.
+
+| Accept Expression | `acceptExpression` | Optional DataWeave expression for a filter that determines whether to process a record in the component. If _both_ the `acceptExpression` _and_ the `acceptPolicy` values for a record evaluate to `true`, then the component accepts the record for processing. If not, the component skips the record, and the record becomes available to any downstream Batch Step components. Examples:
```

Comment:
```suggestion
| Accept Expression | `acceptExpression` | Optional DataWeave expression for a filter that determines whether to process a record in the component. If _both_ the `acceptExpression` _and_ the `acceptPolicy` values for a record evaluate to `true`, then the component accepts the record for processing. If not, the component skips the record, and the record becomes available to any downstream Batch Step components. The following examples illustrate the `acceptExpression`:
```

Reply:
This intro phrase might be too much, but I just don't like the fragment otherwise

Reply:
Gonna keep fragment to facilitate scanning a bit since it's a reference and already quite long.